src/amchart/CommonXY.js

```diff
@@ -214,6 +214,7 @@
             theme: "none",
             type: "xy",
             automargins: false,
+            path: "//www.amcharts.com/lib/3/",
```

Comment:
For some reason, and i spoke with amcharts to ... the AmCharts funnel object cant discover the correct path automatically like it should for the images so this tells it where the base path is so we dont get a 404 on the image load (scrollbar imgs etc...). AmCharts was not able to reproduce with a basic usage of requirejs and loading there lib. But I cant seem to find the root cause ... If you can shed any light i'd welcome a second opinion/option